src/utils/results-reporter.ts

```diff
@@ -85,33 +87,57 @@ export class ResultsReporter {
     const screensWithDifferences = screenshotDiffResults.filter(
       (result) => result.outcome === "diff"
     ).length;
-    const totalScreens = screenshotDiffResults.length;
+    const totalScreensCompared = screenshotDiffResults.length;
+    const totalScreenshotsTaken = testCaseResults.reduce(
+      (total, testCase) => total + testCase.totalNumberOfScreenshots,
+      0
+    );
 
     if (screensWithDifferences === 0) {
       if (!(testRun.project as EnrichedProject).isGitHubIntegrationActive) {
         await this.setCommitStatus({
-          description: `Zero differences across ${totalScreens} screens tested`,
+          description: `Zero differences across ${totalScreensCompared} screens tested`,
           state: "success",
           targetUrl: testRun.url,
         });
       }
-      if (totalScreens > 0) {
+      if (totalScreensCompared > 0) {
         await this.setStatusComment({
           createIfDoesNotExist: true,
-          body: `✅ Meticulous spotted zero visual differences across ${totalScreens} screens tested: [view results](${testRun.url}).`,
+          body: `✅ Meticulous spotted zero visual differences across ${totalScreensCompared} screens tested: [view results](${testRun.url}).`,
         });
+      } else {
+        if (totalScreenshotsTaken === 0) {
+          if (testCaseResults)
+            await this.setStatusComment({
+              createIfDoesNotExist: true,
+              body: `❌ Meticulous replayed ${testCaseResults.length} user sessions, but no screenshots were taken. This likely means there was an error replaying the sessions. Please view the logs of the Github workflow.`,
+            });
+        } else {
+          const baseRefStr = this.options.baseRef
+            ? this.options.baseRef
+            : "main/master";
+
+          // This likely means that the baseRef is not set up for Meticulous yet, so we can't compare against it.
+          // Usually this means that the user has just set up Meticulous and is running it for the first time.
+          await this.setStatusComment({
+            createIfDoesNotExist: true,
+            body: `🤖 Meticulous replayed ${testCaseResults.length} user sessions and took ${totalScreenshotsTaken} screenshots. Meticulous did not run on ${this.options.baseRef} of the ${baseRefStr} branch and so there was nothing to compare against.
```

Comment:
Maybe link to the screenshots? `took ${totalScreenshotsTaken} screenshots` could become a link?

Reply:
https://github.com/alwaysmeticulous/report-diffs-action/pull/220/commits/51c8d3b07b24a75adf6164da2ecbf234b3ff8990 adds a link